tests/layer_tests/README.md

```diff
@@ -0,0 +1,25 @@
+## How to setup
+
+* Set up environment variables for layer tests:
+
+> export MO_ROOT=PATH_TO_MO
+
+> export PYTHONPATH="path_to_openvino"/tests/layer_tests/:$PYTHONPATH
+
+* To compare scoring results need:
+    
+    * Set up additional environment variables:
+    
+        >export IE_APP_PATH="path_to_IE"
+    
+        >export IE_CUSTOM_LAYER="path_to_IE"
+    
+    * Add IE dependencies in LD_LIBRARY_PATH 
+    
+    * To use timelimit tool add path to the tool in PATH
```

Comment:
is timelimit tool used somewhere? Let's remove note from README since it's outdated